### DISCUSSION TITLE
다인님의 코드리뷰, 리드미 작성 가이드

### DISCUSSION THREAD

**Discussion by summer787** [FRONTENDSCHOOL6/react-project-11#117] [Announcements]
## 👨‍👩‍👧‍👦팀원들이 각각 작성해야 할 것
 - 기능 별 코드리뷰
 - 배포 후 LightHouse 검사결과 페이지 (배포 후 예정..?)
 -  리드미 초안 참고
https://github.com/kimInDa/react-project-11/blob/readme/README.md

## 💡코드 리뷰 작성하는 방법

![image](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/90172574/65737e6e-d839-4b91-82de-71536eb88168)

- 각 항목별로 페이지를 만들어 작성할 예정
- 따로 추가적으로 페이지 만들어 작성도 가능
- 마음대로 다시 구성하세요! 
 
##  💡각 항목별로 wiki에 글 작성하는 방법

![image](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/90172574/8580f3d1-6455-4d4f-9e30-f82462f53e57)

### 🚨제목 주의해서 작성해주세요
- [Code Review] 성함 - 기능명
- 이유 
  -  wiki에 작성한 모든 글의 원본이 목록으로 들어와 있습니다
  - 이때 제목의 가나다 순으로 정렬이 되어서 나중에 찾기가 쉽습니다
  -  데일리 스크럼 목록이 원본글의 링크를 연결시켜서 만든 것 입니다
  -  나중에 사이드바에 글 연결 시킬때 찾기쉽습니다
![image](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/90172574/8d564b36-b406-490e-8399-5223df73f5ce)
![image](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/90172574/cda6eb23-2fee-47e7-8c1a-e1d1191b58e6)

  
### wiki는 글 제목을 바꾸면 링크가 달라진다 

 사이드바에 링크를 연결 시킨 후에 글 제목을 수정 시 
 사이드바에 연결한 링크랑 달라져버려서 Pages 에서 원본을 찾아 들어가야 합니다

## 🚨제목만 주의해서 작성해주시면 됩니다!
혹시라도 제목을 바꾸시면 다인님에게 말씀 해주세요!!!

- 각 글은 사이드바 뿐만 아니라 리드미 안에 저 기능 목록에도 연결시킬 예정
- 제목이 달라지면 리드미와 사이드바 주소 수정이 꼭 필요합니다
-  직접 주소 수정을 하셔도 좋습니다. 어떻게 수정하는지 모르겠다 싶으시면 다인님께 말씀만 꼭 부탁드립니다

![image](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/90172574/1803b5e9-38c6-4cf3-af9b-eeb91a7b674d)

## 💡구현한 페이지 전체 스크린샷 찍는 방법
![image](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/90172574/1c9c82e1-2774-4f7d-9606-f65e5a69db97)
 
- 페이지 크기 조절 후
- 요소의 html 에서 마우스 오른쪽 버튼 클릭
- 노드 스크린샷 캡쳐 클릭

**Comment by kimInDa** [comment]
우와 어떻게 이렇게 다 정리하셨어요🥹 송엽님 최고👍

**Reply by summer787** [reply]
다인님이 채팅창에서 자세히 알려주신 부분 다시 참고하고 싶어서 그냥 가져오기만 했습니다! 늘 감사합니다 다인님!! 🥹
